crates/bridge-tx-builder/src/constants.rs

```diff
@@ -30,5 +28,5 @@ lazy_static::lazy_static! {
     /// This is generated via <https://github.com/alpenlabs/unspendable-pubkey-gen> following [BIP 341](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#constructing-and-spending-taproot-outputs)
     /// with `r = 0x82758434e13488368e0781c4a94019d3d6722f854d26c15d2d157acd1f464723`.
     pub static ref UNSPENDABLE_INTERNAL_KEY: XOnlyPublicKey =
-        XOnlyPublicKey::from_str("2be4d02127fedf4c956f8e6d8248420b9af78746232315f72894f0b263c80e81").unwrap();
+        "2be4d02127fedf4c956f8e6d8248420b9af78746232315f72894f0b263c80e81".parse::<XOnlyPublicKey>().expect("valid pubkey");
```

Comment:
Saying "valid pubkey" here seems slightly weird when it'd only ever be printed in the case that it's *invalid*.

Reply:
Agree that `expect` has weird semantics but this is fine at least based on the recommended message style in [std](https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style). Would maybe say, "should be a valid xonly pubkey".

Reply:
Updated expected message.